app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,27 @@
+class InvitesController < ApplicationController
+  before_action :require_user
+
+  def new
+    @invitation = Invite.new
+  end
+
+  def create
+    @invitation = current_user.invites.new(invite_params)
+
+    if @invitation.save
+      @invitation.update_column(:invite_token, SecureRandom.urlsafe_base64)
```

Comment:
This could be extracted to a method like `.generate_token!` and moved to model layer. The other way I would prefer could be a callback hook like `before_save` or `after_create` generating and saving/updating column for token.